src/components/molecules/inputs/Input/stories/input.stories.tsx

```diff
@@ -75,6 +75,10 @@ export const Email = (args: InputProps) => (
   <Input name='Email' type="email" icon='notifications' label="Email" {...args} />
 )
 
+export const Number = (args: InputProps) => (
+  <Input name='Nummer' type="number" icon='euro' label="Nummer" {...args} />
```

Comment:
if this is a currency (icon=euro) i would love to see a matching pattern with comma and points...

Reply:
We could also add a pattern for currencies :O the icon was just randomly chosen :D